book-example/src/for_developers/preprocessors.md

```diff
@@ -29,4 +29,55 @@ pub struct PreprocessorContext {
     pub root: PathBuf,
     pub config: Config,
 }
-```
\ No newline at end of file
+```
+
+## A complete Example
+
+The magic happens within the `run(...)` method of the `Preprocessor` trait implementation.
```

Comment:
It's probably a good idea to add a link to the [`Preprocessor`] trait definition here.

[`Preprocessor`]: https://docs.rs/mdbook/0.1.3/mdbook/preprocess/trait.Preprocessor.html

Reply:
I agree! For now I chose to link to the version directly, without wildcards.